pkg/api/endpoints/util.go

```diff
@@ -28,6 +28,18 @@ import (
 	hashutil "k8s.io/kubernetes/pkg/util/hash"
 )
 
+const (
+	PodHostnameAnnotation  = "net.beta.kubernetes.io/podHostname"
+	PodSubdomainAnnotation = "net.beta.kubernetes.io/podSubdomain"
+	PodHostNamesAnnotation = "net.beta.kubernetes.io/podHostnames"
```

Comment:
done. I called it endpoints, in the plural. 
do you think singular is more appropriate ?